components/status_dropdown/status_dropdown.jsx

```diff
@@ -78,6 +103,25 @@ export default class StatusDropdown extends React.PureComponent {
         this.props.actions.openModal(resetStatusModalData);
     };
 
+    showCustomStatusModal = () => {
+        const customStatusInputModalData = {
+            ModalId: ModalIdentifiers.CUSTOM_STATUS,
+            dialogType: CustomStatusModal,
+            dialogProps: {userId: this.props.userId},
+        };
+
+        this.props.actions.openModal(customStatusInputModalData);
+    }
+
+    showCustomStatusTextTooltip = () => {
+        const element = this.customStatusTextRef;
+        if (element && element.offsetWidth < element.scrollWidth) {
+            this.setState({showCustomStatusTooltip: true});
+        } else {
+            this.setState({showCustomStatusTooltip: false});
+        }
+    }
```

Comment:
this can be simplified since the condition in the if-clause basically resembles the status that is set in the `setState`, so it could look like this:

```suggestion
    showCustomStatusTextTooltip = () => {
        const element = this.customStatusTextRef;
        this.setState({showCustomStatusTooltip: element && element.offsetWidth < element.scrollWidth});
    }
```